### PR TITLE
Configure: warn when 'none' is the chosen seed source

### LINKED ISSUE #6980: Add a warning when using NONE as a seed source
It would be prudent to add a warning to Configure about using NONE as a seed source.

Issue reported by OSTIF.

### PR DESCRIPTION
**Pull request by levitte**
Fixes #6980

### CODE REVIEW

File: Configure [hunk 1]
```diff
@@ -1012,9 +1012,13 @@ if (scalar(@seed_sources) == 0) {
...
     push @seed_sources, 'os';
 }
-die "Cannot seed with none and anything else"
-    if scalar(grep { $_ eq 'none' } @seed_sources) > 0
-        && scalar(@seed_sources) > 1;
+if (scalar(grep { $_ eq 'none' } @seed_sources) > 0) {
+    die "Cannot seed with none and anything else" if scalar(@seed_sources) > 1;
+    warn <<_____ if scalar(@seed_sources) == 1;
+You have selected NO seed source.  Some functions (such as creating keys)
+will not work.
```
Comment (mattcaswell) [inline Configure]: Well they won't work without manual seeding. Perhaps a pointer towards the RAND_DRBG(7) page?

Comment (mspncp) [reply Configure]: My humble suggestion:

``` 
You have selected the --with-rand-seed=none option, which effectively disables
automatic reseeding of the OpenSSL random generator. All operations depending on
the random generator such as creating keys will not work unless the random generator
is seeded manually by the application.

Please read the 'Note on random number generation' section in the INSTALL instructions 
document and the RAND_DRBG(7) manual page for more details.
```

Comment (mspncp) [reply Configure]: correction: instructions ~~document~~

**Review by mspncp — APPROVED** [review]
LGTM

**Comment by levitte** [comment]
Merged.

2805ee1e095a78f596dc7adf778441e2edb9f15c Configure: warn when 'none' is the chosen seed source

**Pull request closed without merging**